src/main/java/gregtech/integration/jei/recipe/GTRecipeWrapper.java

```diff
@@ -107,6 +126,8 @@ public void addItemTooltip(int slotIndex, boolean input, Object ingredient, List
             tooltip.add(I18n.format("gregtech.recipe.chance", chance, boost));
         } else if (notConsumed) {
             tooltip.add(I18n.format("gregtech.recipe.not_consumed"));
+        } else if (recipe.getOutputs().get(0).hasTagCompound() && ((ItemStack) ingredient).isItemEqual(new ItemStack(recipe.getOutputs().get(0).getSubCompound("asslineOutput")))) {
+            tooltip.addAll(Arrays.asList(I18n.format("gregtech.recipe.assline_scanner").split("\\\\n")));
```

Comment:
Somewhere in CE there is something that auto splits lang entries when detecting a `\n` character. You could use that instead of calling split

Reply:
useless now that i slimmed it down to one line